govcd/system_test.go

```diff
@@ -219,3 +219,17 @@ var INVALID_NAME = `*******************************************INVALID
 func init() {
 	testingTags["system"] = "system_test.go"
 }
+
+func (vcd *TestVCD) Test_QueryOrgVdcNetworkByName(check *C) {
```

Comment:
Running: TestVCD.Test_QueryOrgVdcNetworkByName
system_test.go:232:
    check.Assert(len(orgVdcNetwork), Not(Equals), 0)
... obtained int = 0
... expected int = 0

FAIL: system_test.go:223: TestVCD.Test_QueryOrgVdcNetworkByName

Reply:
Fixed